src/pydata_sphinx_theme/theme/pydata_sphinx_theme/components/navbar-logo.html

```diff
@@ -1,19 +1,31 @@
-{% if logo %}
 {% if not theme_logo_link %}
-<a class="navbar-brand" href="{{ pathto(root_doc) }}">
-  <img src="{{ pathto('_static/' + logo, 1) }}" class="logo" alt="logo">
-</a>
-{% elif theme_logo_link[:4] == 'http' %}
-<a class="navbar-brand" href="{{ theme_logo_link }}">
-  <img src="{{ pathto('_static/' + logo, 1) }}" class="logo" alt="logo">
-</a>
+  {% set href = pathto(root_doc) %}
+{% elif theme_logo_link[:4] == "http" %}
+  {% set href = theme_logo_link %}
 {% else %}
-<a class="navbar-brand" href="{{ pathto(theme_logo_link) }}">
+  {% set href = pathto(theme_logo_link) %}
+{% endif %}
+
+{% if theme_dark_logo and theme_light_logo %}
+
+  <a class="navbar-brand only-light" href="{{ href }}">
+    <img src="{{ pathto('_static/' + theme_light_logo, 1) }}" class="logo" alt="logo">
+  </a>
+
+  <a class="navbar-brand only-dark" href="{{ href }}">
```

Comment:
I think that we could actually just have a single `<a class="navbar-brand" href="{{ href }}">` container in this template, and inside that link we could do the conditional check for whether to use `img` or `p` based on the configuration (and then we could use `only-dark` and `only-light` for the `<img` tags instead of the `<a` tag

Reply:
done it's now also solving #545

Reply:
done it's now also solving https://github.com/pydata/pydata-sphinx-theme/issues/545